File: CinemaProject.WebAPI/CinemaProject.BLL/Models/Cinema.cs
using System;

namespace CinemaProject.BLL.Models
{
    public class Cinema
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Guid CityId { get; set; }

        public Hall[] Halls { get; set; }
    }
}

File: CinemaProject.Angular/src/app/AdminPages/Models/cinema.ts
import { Hall, HallView } from '../Models/hall';
import { TypeOfSeat, TypeOfSeatView } from './typeOfSeat';

class Cinema {
  constructor(
    public id: string,
    public name: string,
    public cityId: string,
    public halls: Hall[] | null,
    public typesOfSeat: TypeOfSeat[] | null
  ) {}
}

class CinemaView {
  constructor(
    private id: string,
    public name: string,
    public halls: HallView[],
    public typesOfSeat: TypeOfSeat[],
    public cityName?: string
  ) {}
}

export { Cinema, CinemaView };
